MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -89,39 +89,21 @@ private static void ProcessItemCommand(GameSession session, string command)
                 return;
             }
 
-            // Add some bonus attributes to equips and pets
-            ItemStats stats = new ItemStats();
-            if (ItemMetadataStorage.GetTab(itemId) == InventoryTab.Gear
-                    || ItemMetadataStorage.GetTab(itemId) == InventoryTab.Pets)
-            {
-                Random rng = new Random();
-                stats.BonusAttributes.Add(ItemStat.Of((ItemAttribute) rng.Next(35), 0.01f));
-                stats.BonusAttributes.Add(ItemStat.Of((ItemAttribute) rng.Next(35), 0.01f));
-            }
-
-            Item item = new Item(itemId)
+            if (int.TryParse(config.GetValueOrDefault("rarity", "5"), out int rarity))
+            { }
```

Comment:
Why is this in an empty if statement?

Reply:
Editor requested TryParse to check if conversion was successful. Maybe I can use discard instead.
`_ = int.TryParse`

Reply:
That's probably better tbh